tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -132,8 +132,21 @@ Verify Container Image
     Should Be Equal  ${image}  ${expected-image-url}
 
 Search Pod
-    [Documentation]   Return list of pods   pod_start_with = here ypu have to provide starting of pod name
+    [Documentation]   Returns list of pods   pod_start_with = here ypu have to provide starting of pod name
     [Arguments]   ${namespace}  ${pod_start_with}
     ${pod} =  Run  oc get pods -n ${namespace} -o json | jq '.items[] | select(.metadata.name | startswith("${pod_start_with}")) | .metadata.name'
     @{list_pods} =  Split String  ${pod}  \n
     [Return]  ${list_pods}[0]
+
+Run Command In Container
+    [Documentation]    Executes a command in a container.
+    ...    If ${container_name} is omitted, the first container in the pod will be chosen
+    [Arguments]    ${namespace}    ${pod_name_start_with}    ${command}    ${container_name}=${EMPTY}
+    ${pod_name}    Search Pod    namespace=${namespace}    pod_start_with=${pod_name_start_with}
```

Comment:
I still don't understand if this works on a single pod or a list of pods. And if it works on a list of pods, on what container/pod is it actually running the command?

Reply:
It works on a single pod(Which is provided by `Search Pod` )
if we provide the name of the container then it will exec the command on that container.
if it is not provided then it will exec command on first container or we can say on default container
I will share example with you in googlechat

Reply:
I think `Search Pod` returns just the first pod matching the name. Rushikesh has updated the [Documentation] tag to reflect that (he didn't change the behaviour)

Reply:
Yes @jgarciao 
I have changed documentation,not behavior

Reply:
@jgarciao as I wrote in the other comment, I don't like the fact that `Search Pod` arbitrarily returns the first pod out of a list of an unknown number of pods; I feel like this is not a good functionality, and that sooner or later someone is going to try to reuse that keyword and it won't work. When that happens, we will either have to replace the keyword or update the internal logic, which might lead to this TC then breaking apart. 
IMHO we should write the keyword in such a way that `Search Pod` returns *exactly* the pod you want, not just the first one according to `oc`'s ordering of the list, but if you feel that this is ok I've already given the approval to the PR and we can proceed with the merge.